src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/ApprovedPremisesRoomsSeedFromXLSXJob.kt

```diff
@@ -0,0 +1,210 @@
+    "Does this room have en-suite bathroom facilities?" to "hasEnSuite",
+    "Are corridors leading to this room of sufficient width to accommodate a wheelchair? (at least 1.2m wide)" to "isWheelchairAccessible",
+    "Is the door to this room at least 900mm wide?" to "hasWideDoor",
+    "Is there step free access to this room and in corridors leading to this room?" to "hasStepFreeAccess",
+    "Are there fixed mobility aids in this room?" to "hasFixedMobilityAids",
+    "Does this room have at least a 1500mmx1500mm turning space?" to "hasTurningSpace",
+    "Is there provision for people to call for assistance from this room?" to "hasCallForAssistance",
+    "Can this room be designated as suitable for wheelchair users?   Must answer yes to Q23-26 on previous sheet and Q17-19 & 21 on this sheet)" to "isWheelchairDesignated",
+    "Can this room be designated as suitable for people requiring step free access? (Must answer yes to Q23 and 25 on previous sheet and Q19 on this sheet)" to "isStepFreeDesignated",
+//    these characteristics are not in the site survey data but are in the database
+//    "Is this an IAP?" to "isIAP",
+//    "Is this AP a PIPE?" to "isPIPE",
+//    "Is this AP an Enhanced Security AP?" to "isESAP",
+//    "Is this AP semi specialist Mental Health?" to "isSemiSpecialistMentalHealth",
+//    "Is this a Recovery Focussed AP?" to "isRecoveryFocussed",
+//    "Is this AP suitable for people at risk of criminal exploitation? N.B Enhanced Security sites answer No, other APs answer Yes." to "isSuitableForVulnerable",
+//    "Does this AP accept people who have committed sexual offences against adults?" to "acceptsSexOffenders",
+//    "Does this AP accept people who have committed sexual offences against children?" to "acceptsChildSexOffenders",
+//    "Does this AP accept people who have committed non-sexual offences against children?" to "acceptsNonSexualChildOffenders",
+//    "Does this AP accept people who have been convicted of hate crimes?" to "acceptsHateCrimeOffenders",
+//    "Is this AP Catered (no for Self Catered)?" to "isCatered",
+//    "Is there a step free entrance to the AP at least 900mm wide?" to "hasWideStepFreeAccess",
+//    "Are corridors leading to communal areas at least 1.2m wide?" to "hasWideAccessToCommunalAreas",
+//    "Do corridors leading to communal areas have step free access?" to "hasStepFreeAccessToCommunalAreas",
+//    "Does this AP have bathroom facilities that have been adapted for wheelchair users?" to "hasWheelChairAccessibleBathrooms",
+//    "Is there a lift at this AP?" to "hasLift",
+//    "Does this AP have tactile & directional flooring?" to "hasTactileFlooring",
+//    "Does this AP have signs in braille?" to "hasBrailleSignage",
+//    "Does this AP have a hearing loop?" to "hasHearingLoop",
+//    "Are there any additional restrictions on people that this AP can accommodate?" to "additionalRestrictions",
```

Comment:
I _think_ these are all premise characteristics? In which case they're safe to ignore and can probably remove this comment

Reply:
Removed these commented characteristics now.